pkg/ocibundle/tools/oci.go

```diff
@@ -35,6 +35,14 @@ func (v Volumes) Path() string {
 	return filepath.Join(string(v), "volumes")
 }
 
+// Layers is the parent layers path
```

Comment:
I find the naming in lines 38-45 to be confusing... It looks like the `Layers` object is _not_ actually the "layers" but more like something like `LayersParent`(?), such that what is currently called `Layers.Path()` might be better named something like `LayersParent.LayersPath()`...?

(I see that this follows the convention laid out by `Volumes` - though in isolation, that might just be an argument for fixing _both_ sets of terms - and also that this code all exists under `pkg/`, which might be an argument in favor of leaving it all as it is. So just putting this all here for posterity / as food for thought.)

Reply:
Yeah.. this is another one I wasn't sure of w.r.t. clarity vs. consistency of what's there given limitations of changing things in pkg/

I'll have a quick think again.

Reply:
Tried to resolve via comments for now (as in DM on Slack)... with tech debt issue for the future to give a more sensible API for this stuff.

Reply:
See https://github.com/sylabs/singularity/issues/2304